tasks/pkg-debian.yml

```diff
@@ -42,7 +42,7 @@
     - key: 5F1E256061D813B125E156E8E6266D4AC0962C7D
       value: "{{ datadog_apt_key_url_new }}"
     - key: D18886567EABAD8B2D2526900D826EB906462314
-      value: "{{ datadog_apt_key_url_new }}"
+      value: "{{ datadog_apt_key_url_new }}"
```

Comment:
👀 what's the diff here?

Reply:
Don't ask me, ask `ansible-lint` I just copy pasted what fix he suggested to me 
![idk](https://media0.giphy.com/media/l3vRkGgMfvANIFtE4/giphy.gif?cid=6c09b952m388kr7iq9i74ciu01mh9tt8dezgra0f2gu9ajx8&ep=v1_gifs_search&rid=giphy.gif&ct=g)

Reply:
Non breaking space VS rebular space

Reply:
\t \s ?